osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -3304,6 +3304,33 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/muse.png"
       }
+    },
+    {
+      "description": "The native staking and governance coin of the Unification Blockchain.",
+      "denom_units": [
+        {
+          "denom": "ibc/608EF5C0CE64FEA097500DB39657BDD36CA708CC5DCC2E250A024B6981DD36BC",
+          "exponent": 0,
+          "aliases": ["nund"]
+        },
+        {
+          "denom": "fund",
+          "exponent": 9
+        }
+      ],
+      "base": "ibc/608EF5C0CE64FEA097500DB39657BDD36CA708CC5DCC2E250A024B6981DD36BC",
+      "name": "Unification",
+      "display": "FUND",
```

Comment:
please chg display value: 'FUND' -> 'fund'. it should be lower case to be able to reference the lowercase unit defined under 'denom_units'.